kong/pdk/response.lua

```diff
@@ -536,10 +532,16 @@ local function new(self, major_version)
     local json
     if type(body) == "table" then
       if is_grpc then
-        if type(body.message) == "string" then
+        if is_grpc_output then
```

Comment:
Won't this catch part of the cases where `body.message` should be returned in the header - i.e., should we move the following `elseif` up?

Reply:
@gszr we decided with @hishamhm that if there is response content type given for grpc output (either in exit headers orngx.header), we don't support this magic table. magic table is used only for this compatibility thing, e.g. currently there ia no such plugin that asks grpc response type but there are pkugins and core code tgat sends { message = <string> }.

Reply:
is_grpc_output = send body in body